jbpm-flow-builder/src/main/java/org/jbpm/osgi/flow/compiler/Activator.java

```diff
@@ -35,7 +36,8 @@ public void start(BundleContext bc) throws Exception {
         this.processBuilderReg = bc.registerService( new String[]{ ProcessBuilderFactoryService.class.getName(), Service.class.getName()},
                                                                    new ProcessBuilderFactoryServiceImpl(),
                                                                    new Hashtable() );
-        ProcessBuilderFactory.reInitializeProvider();
+        RuntimeManagerFactory.Factory.reset();
+        ProcessBuilderFactory.reInitializeProvider();        
```

Comment:
not really as they do different things. RuntimeManagerFactory ... reset() does only reset of the flag and exception without reinitializing the factory impl itself